.circleci/images/primary/Dockerfile-2.1.10

```diff
@@ -50,27 +127,14 @@ RUN set -ex \
   && (docker version || true)
 
 # Install Docker Compose
-RUN COMPOSE_URL="https://circle-downloads.s3.amazonaws.com/circleci-images/cache/linux-amd64/docker-compose-latest" \
-  && curl --silent --show-error --location --fail --retry 3 --output /usr/bin/docker-compose $COMPOSE_URL \
-  && chmod +x /usr/bin/docker-compose \
-  && docker-compose version
+RUN echo 'deb http://ftp.debian.org/debian stretch-backports main\ndeb-src http://ftp.debian.org/debian stretch-backports main' > /etc/apt/sources.list.d/stretch-backports.list \
+  && apt-get update \
+  && apt-get install -y docker-compose \
+  && docker-compose version \
+  && rm -rf /var/lib/apt/lists/*
```

Comment:
I'm curious, did the `docker-compose` version we use on the latest images from `https://github.com/linuxserver/docker-docker-compose/releases/download/1.29.2-ls51/` not work on this debian image? (Worth perhaps leaving a comment here explaining why this is different?)

Reply:
Sadly the tarball you mention vendors a python whose glibc shared lib version check fails against the Stretch provided one, resulting in python being un-runnable.

Since stretch-backports has docker-compose at 1.26 (IIRC) instead of 1.29 I wanted to first test if it worked at all.

Reply:
Ah, I see! I guess my suggestion is to drop a comment explaining it :)

Reply:
Looks like it works! Adding the comment :)